gui/default/untrusted/syncthing/device/editDeviceModalView.html

```diff
@@ -158,7 +162,7 @@
       <span class="fas fa-times"></span>&nbsp;<span translate>Close</span>
     </button>
     <div ng-if="editingExisting" class="pull-left">
-      <button type="button" class="btn btn-warning btn-sm disabled" ng-if="willBeReintroducedBy" tooltip data-original-title="This device will be reintroduced by {{ willBeReintroducedBy }}">
+      <button type="button" class="btn btn-warning btn-sm" ng-if="willBeReintroducedBy" tooltip data-original-title="{{ willBeReintroducedBy }} might reintroduce this device.">
```

Comment:
... as well as in all the tooltips, though I see it also wasn't translated previously. Perhaps we never translate in tooltips because it doesn't work? Or we use the ` | translate` angular thing?

Reply:
See previous comment. But there's still a problem with the interpolation.